src/app/main/component/events/components/event-editor/components/create-event-dates/components/date-time/date-time.component.html

```diff
@@ -0,0 +1,47 @@
+<div [formGroup]="form" class="date-time">
```

Comment:
new component for this part of day 
![image](https://github.com/ita-social-projects/GreenCityClient/assets/73541834/faf5c0df-feb5-4280-8a8c-3bb2eda67d63)